Makefile

```diff
@@ -3,6 +3,7 @@ help:
 	@echo 'docs                   - build and display docs'
 	@echo 'ci_settings            - create a lego/settings/local.py for ci'
 	@echo 'fixme                  - Fix code formatting'
+	@echo 'me-a-devenv-pls        - yes. It does that.'
```

Comment:
rename to devenv

Reply:
Just feels like it comes with way more guarantees if the name isn't silly

Reply:
But I guess I can rename